src/app/app-routing.module.ts

```diff
@@ -4,7 +4,12 @@ import { RouterModule, Routes } from '@angular/router';
 const routes: Routes = [
   {
     path: '',
-    loadChildren: () => import('./home/home.module').then(m => m.HomeModule)
+    loadChildren: () => import('./home/home.module').then(m => m.HomeModule),
+    pathMatch: 'full'
+  },
+  {
+    path: '',
```

Comment:
I would put here the path: 'sessions' (with and s to respect the [restful resource naming](https://restfulapi.net/resource-naming/) ) then in the videoSession module only `:videoId`

Reply:
@Mubramaj 

Fixed, with an `s` (sessions) 😉 

_FYI, Restful resource naming specs are for Restful API, means the BackEnd endpoints, not page URLs_

Reply:
:D I know I couldn't find a doc for URLs sorry, I like applying same rules in the URLs because then they are meaningful and easily understandable

Reply:
no problem @Mubramaj  😄 
Yes, they looks meaningful for developers, but in fact the only thing that matter for URL structure is SEO and what search engine loves 😉

Reply:
Oh interesting, I didn't know that about SEO I am going to look more into it ! Thanks ! ;)